Lib/asyncio/base_events.py

```diff
@@ -717,6 +719,8 @@ def call_at(self, when, callback, *args, context=None):
 
         Absolute time corresponds to the event loop's time() method.
         """
+        if when is None:
+            raise TypeError("when cannot be None")
```

Comment:
Why test only for None? What about other invalid types?

Reply:
The same as above: provide descriptive error message for obvious mistakes, raise default errors for all other combinations.